### PR TITLE
[driver] Fix wrong WS passed to Backward and WrW Find() calls.

### PR DESCRIPTION
**Pull request by atamazov**
Fixes issue #264 originated from #172. Resizing of WS to suit needs of Find() calls was missing.

I need this asap to resume performance/correctness testing that I do from time to time with certain PRs (for ex. #233 and #293 are awaiting this right now)

### CODE REVIEW

File: driver/conv_driver.hpp [hunk 1]
```diff
@@ -1944,9 +1953,11 @@ int ConvDriver<Tgpu, Tref>::FindBackwardData(int& ret_algo_count,
 template <typename Tgpu, typename Tref>
 int ConvDriver<Tgpu, Tref>::FindBackwardWeights(int& ret_algo_count,
                                                 int request_algo_count,
-                                                std::vector<miopenConvAlgoPerf_t>& perf_results)
+                                                std::vector<miopenConvAlgoPerf_t>& perf_results,
+                                                context_t ctx)
 {
     wrw_auxiliary.resume(wall_enabled);
+    ResizeWorkspaceDev(ctx, ws_sizeof_find_wrw);
```
Comment (TejashShah) [inline driver/conv_driver.hpp:1960]: Is this missing func invocation at line no.1933 and 1960 the real cause of why wrong ws size was computed and passed causing the #238?

File: driver/conv_driver.hpp [hunk 2]
```diff
@@ -1919,9 +1926,11 @@ int ConvDriver<Tgpu, Tref>::RunForwardCPU()
 template <typename Tgpu, typename Tref>
 int ConvDriver<Tgpu, Tref>::FindBackwardData(int& ret_algo_count,
                                              int request_algo_count,
-                                             std::vector<miopenConvAlgoPerf_t>& perf_results)
+                                             std::vector<miopenConvAlgoPerf_t>& perf_results,
+                                             context_t ctx)
 {
     bwd_auxiliary.resume(wall_enabled);
+    ResizeWorkspaceDev(ctx, ws_sizeof_find_bwd);
```
Comment (TejashShah) [inline driver/conv_driver.hpp:1933]: Is this missing func invocation at line no.1933 and 1960 the real cause of why wrong ws size was computed and passed causing the #238?

File: driver/conv_driver.hpp (hunk 1 above)
Comment (atamazov) [reply driver/conv_driver.hpp:1960]: I do not know what us the real cause of #238. But this fix is definitely required.

**Comment by atamazov** [comment]
This blocks my work.

**Pull request merged**